src/cards/stats-card.js

```diff
@@ -203,13 +185,22 @@ const renderStatsCard = (stats = {}, options = { hide: [] }) => {
     return measureText(custom_title ? custom_title : i18n.t("statcard.title"));
   };
 
-  const width = hide_rank
+  /*
+    When hide_rank=true, minimum card width is max of 270 and length of title + paddings.
+    When hide_rank=false, minimum card_width is 340.
+    Numbers are picked by looking at existing dimensions on production.
+  */
+  const minCardWidth = hide_rank
     ? clampValue(
         50 /* padding */ + calculateTextWidth() * 2,
-        270 /* min */,
-        Infinity,
-      )
-    : 495;
+        270,
+        Infinity)
+    : 340
```

Comment:
I think here you should also take into account the icon width when the show_icons flag is set to true. This is 16 px. Please merge https://github.com/postatum/github-readme-stats/pull/1 to fix this issue.